src/CustomMimetypeHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Http\Message\MultipartStream;
+
+/**
+ * Let you add your own mimetypes.
+ *
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+class CustomMimetypeHelper extends ApacheMimetypeHelper
+{
+    /**
+     * @var array
+     */
+    private $mimetypes = [];
+
+    /**
+     * @param array $mimetypes
+     */
+    public function __construct(array $mimetypes = [])
+    {
+        $this->mimetypes = $mimetypes;
+    }
+
+    /**
+     * @param string $extension
+     * @param string $mimetype
+     *
+     * @return $this
+     */
+    public function addMimetype($extension, $mimetype)
```

Comment:
Im not sure this would ever be used, no. But I thought "why not?" This class should only be used in the 1% edge case. And if you are such end case you are probably doing something special and could probably be in a scenario when you want to edit this at runtime.